Automattic/jetpack
# RNA: IDC package layout

#### Changes proposed in this Pull Request:
* create the new RNA IDC package
* build the basic layout for the main screen
* modify the Connection UI package to make it always display the IDC screen for debugging purposes

#### Jetpack product discussion
p9dueE-3kA-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Go to "Tools -> Connection Manager".
2. Confirm that you see the "Safe Mode" screen and that it looks pretty much like the screenshot below, and centered in the middle of the screen.
3. Try to change the screen size, confirm that it still looks good.
4. Click on the "Move your settings" and "Create a fresh connection" buttons. They are not functional, but should display the spinner in the button for 3 seconds upon click, confirm that it happens.

#### The screenhot
<img width="1160" alt="Screen Shot 2021-09-22 at 6 36 23 PM" src="https://user-images.githubusercontent.com/1341249/134431126-9edf3209-f3f3-4522-82af-89c66c82faac.png">
